packages/fxa-payments-server/src/store/index.tsx

```diff
@@ -4,6 +4,7 @@ import ReduxThunk, { ThunkMiddleware } from 'redux-thunk';
 import { createPromise as promiseMiddleware } from 'redux-promise-middleware';
 import { reducers } from './reducers';
 import { State, Action } from './types';
+import { AmplitudeEvent } from './amplitude-event';
```

Comment:
Just a suggestion: I'd call this `amplitudeMiddleware` or even `userMetricsMiddleware`, to emphasize that it's a middleware.